uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/util/GWTEditorNativeRegister.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.client.util;
+
+import org.jboss.errai.ioc.client.container.SyncBeanDef;
+import org.uberfire.client.mvp.Activity;
+
+/**
+ *  Call native Javascript to register GWT Editors
+ */
+public class GWTEditorNativeRegister {
```

Comment:
I would make this class an @ApplicationScoped bean and inject it where needed. Makes it easier to mock when testing.